src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -122,10 +124,10 @@ export default function VideoVisitLocation({ appointment, facility }) {
           {!!facility && (
             <span className="vads-u-display--block vads-u-margin-top--2">
               {name}
-              {phone && (
```

Comment:
This current phone value is dependent on logic in the calendar data generation that's related to when we show the number in the ics file. That's not the same logic for when to show the phone here.